build.gradle.kts

```diff
@@ -114,6 +114,8 @@ recipeDependencies {
 
     parserClasspath("org.apache.httpcomponents.core5:httpcore5:5.1.+")
     parserClasspath("org.apache.httpcomponents.client5:httpclient5:5.1.+")
+
+    parserClasspath("org.apache.tomcat.embed:tomcat-embed-core:10.1.+")
```

Comment:
Is there a smaller tomcat library we could add here? One with just the classes we need for type attribution, not everything needed to run tomcat?

Reply:
We only really need the jakarta servlet api, will update this